node/pkg/governor/governor_prices.go

```diff
@@ -2,7 +2,7 @@
 //
 // The initial prices are read from the static config (tokens.go). After that, prices are
 // queried from CoinGecko. The chain governor then uses the maximum of the static price and
-// the latest CoinGecko price.
+// the latest CoinGecko price. CoinGecko is polled every five minutes.
```

Comment:
If you made the polling interval  a constant this comment wouldn't be necessary. Maybe?

```
const coinGeckoQueryIntervalMins = 5
```